blog/_posts/2021-10-06-manual-week.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "ROOT Manual's week"
+layout: archive
+author: Olivier Couet
+---
+
+ROOT has now a brand new _[Manual]({{ '/manual' | relative_url }})_, bringing
+_"how do you read data with ROOT"_ and similar core aspects into contemporary ROOT.
+Much of the manual has been rethought and simplified, we hope and expect that concepts and
+interfaces are now easier to understand !
+
+
+To achieve this, all the ROOT team was involved in a big update of the
+_[Manual]({{ '/manual' | relative_url }})_ and _[Reference Guide](https://root.cern/doc/master/index.html)_
+during one full week.
+
+Previously, the ROOT documentation was spread over three different main manuals:
+
+  - The _[Reference Guide](https://root.cern/doc/master/index.html)_
+  - The _[Manual]({{ '/manual' | relative_url }})_
+  - The _Old ROOT User’s Guide_
+
+The _[Reference Guide](https://root.cern/doc/master/index.html)_ and the
+_[Manual]({{ '/manual' | relative_url }})_ are the current valid sources of documentation.
+_[The Manual]({{ '/manual' | relative_url }})_ acts as a "User's Guide" helping users to find their way into the huge amount of documentation
+provided in the _[Reference Guide](https://root.cern/doc/master/index.html)_.
+The _Old User's Guide_ is outdated and not updated but, nevertheless it contains some valuable
+information we do not want to lose.
```

Comment:
```suggestion
The _Old User's Guide_ was outdated and not updated but nevertheless contained some valuable
information we did not want to lose.
```
(past tense to clarify that we don't think it's needed anymore after our "intervention")